moonshot/src/api/api_session.py

```diff
@@ -57,9 +88,10 @@ def api_get_available_session_info() -> tuple[list, list]:
     runner_with_session_db_list = []
 
     for runner_instance in runner_instances:
-        if Session.load(runner_instance.database_instance) is not None:
-            runner_ids.append(runner_instance.id)
-            runner_with_session_db_list.append(runner_instance.database_instance)
+        if runner_instance.database_instance:
+            if Session.load(runner_instance.database_instance) is not None:
+                runner_ids.append(runner_instance.id)
+                runner_with_session_db_list.append(runner_instance.database_instance)
```

Comment:
just curious what is going to happen if the database instance is None or session is None

Reply:
if a database instance is None, that particular runner will be skipped.
if a Session is None, that particular runner will be skipped as well. This should not be a None as if there is no Session detected in the runner, one will be created but a None will be returned if there is an error creating the Session.
if all are None, empty lists will be returned